README.md

```diff
@@ -30,6 +30,10 @@ Installation
 
 [![Get Freeciv21 from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-white.svg)](https://snapcraft.io/freeciv21)
 
+### Flathub Edition
+
+Freeciv21 is also available on [Flathub](https://flathub.org/apps/net.longturn.freeciv21).
```

Comment:
What about using https://flathub.org/badges next to the Snap one?

Reply:
Oh, didn't know that existed. Good catch. I'll do another commit.